pytradfri/device.py

```diff
@@ -263,7 +263,7 @@ def _value_validate(self, value, rnge, identifier="Given"):
         """
         Make sure a value is within a given range
         """
-        if value < rnge[0] or value > rnge[1]:
+        if value is not None and (value < rnge[0] or value > rnge[1]):
```

Comment:
Could we not have this as;

`if value is None or (value < rnge[0] or value > rnge[1]):`

Reply:
That would raise the error if `value == None`. Here we never want `None` to raise an error.